source/shared/net/desktop-wallet/change-restake-settings.rst

```diff
@@ -9,9 +9,13 @@ By default, your baker rewards are restaked when you create the baker account. H
 Change restake settings (Single-signature account)
 ==================================================
 
-#. Go to **Accounts**, select the account whose baker keys you want to update, change the view to the detailed view, and select **Update baker restake earnings**.
+#. Go to **Accounts** and select the account whose baker restake preferences you want to change.
 
-#. Select whether you want to restake earnings or not, and then select **Continue**.
+#. Click **More options** and select **Baking**.
+
+#. Click **Update baker stake**.
+
+#. You can change the staked amount, if desired. Select whether you want to restake earnings or not, and then select **Continue**.
```

Comment:
Since this page is not about updating the staked amount, probably there should be a caveat here about cool-down with respect to reducing the staked amount, and/or a link to the relevant section of the "update the staked amount" page. (And similarly for multi-sig.)